src/lib/device.js

```diff
@@ -127,10 +127,9 @@ export function isIECompHeader() : boolean {
 }
 
 export function isElectron() : boolean {
-    if (typeof process !== 'undefined' && process.versions && process.versions.electron) {
-        return true;
-    }
-    return false;
+    let userAgent = getUserAgent();
```

Comment:
Can we also keep the `process.versions.electron` check, for backwards-compat? We can do an `||` on that and the user-agent.

Reply:
Actually I think the previous check was not relevant at all so I'm not sure it can provide any backwards-compatibility. Regarding other versions, I've just quickly looked through the aggregated data grouped by different electron versions (https://developers.whatismybrowser.com/useragents/explore/software_name/electron-application/1) and came to conclusion that the current check will pass in every case.